docs/com.bitwarden.organizations.md

```diff
@@ -0,0 +1,79 @@
+[Table of contents](README.md#table-of-contents)
+
+# Bitwarden organization
+
+The `com.bitwarden.organization` doctype is used to group
+[ciphers](./com.bitwarden.ciphers.md) in a single unit and/or to share them with
+other [bitwarden contacts](./com.bitwarden.contacts.md).
+
+## Attributes
+
+- `name`: {string} - the name of the organization
+- `members`: {object} - the members of the organization (see below)
+- `defaultCollection`: {object} - the collection of the organization (see below)
+- `cozyMetadata`: {object} - the [cozyMetadata](README.md#document-metadata)
+
+## Members
+
+An organization can be shared with a list of members.
+
+- `email`: {string} - the email of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+- `key`: {string} - the public key of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+- `name`: {string} - the name of the member
+- `owner`: {bool} - true if the member owns the organization
+- `status`: {int} - the status of the member (`0` = `invited`, `1` = `accepted`, `2` = `confirmed`)
+- `user_id`: {string} - the id of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+
+Note that the `name` and `key` attributes may be missing if the member status is still `invided` as those data are sent only when he accepts the first [Cozy sharing](./io.cozy.sharings.md) from organization's owner.
```

Comment:
Don't we also need the URL somewhere?

Reply:
s/invided/invited

Reply:
Typo fixed.

What URL are you talking about? The URL to the Cozy instance or something else?

Reply:
Yes, the URL of the member's Cozy instance, but I guess we actually save this info in the `io.cozy.sharings.md`doc? Thus, the members are duplicated between `io.cozy.sharings.md` and `com.bitwarden.organizations.md `?

Reply:
Ok. So yes it is saved on Cozy's doctypes. Bitwarden protocol wouldn't need or use that URL. I guess minimal requirements are user ID and Key and that other data would be displayed on organization management's page but this page will not be implemented on Cozy's solution (as this is handled by cozy-sharing's dialog).